### PR TITLE
dita-ot 3.1 (new formula)

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
@fxcoudert @ilovezfs got any more of them reviews?

File: Formula/dita-ot.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  def install
+    rm_f Dir["bin/*.bat", "startcmd.*"]
+    rm_rf "temp"
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"bin/dita" => "dita"
+  end
+
+  test do
+    assert_match "DITA-OT version #{version}", shell_output("#{bin}/dita --version")
```
Comment (ilovezfs) [inline Formula/dita-ot.rb]: would like to see a functional test here

**Review by ilovezfs — DISMISSED** [review]

File: Formula/dita-ot.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/dita-ot.rb]: Done, it's now parsing the documentation that's part of every release.

**Comment by ilovezfs** [comment]
Thanks again @SMillerDev! 🍭

**Pull request merged**